examples/platforms/da15000/radio.c

```diff
@@ -413,7 +416,7 @@ void FTDF_sendFrameTransparentConfirm(void         *handle,
 void da15000RadioProcess(otInstance *aInstance)
 {
     bool rxPending;
-    uint8_t value = 0;
+    uint8_t bufferFor_FTDF_configValue = 0;
```

Comment:
I can't seam to find any mention of this in any style guides.  Most seam to advise to go one way or the other, but not to mix.
Linux Kernel, GNU, and Google say snake case only:
https://www.gnu.org/prep/standards/html_node/Names.html#Names
https://google.github.io/styleguide/cppguide.html#Variable_Names
https://www.kernel.org/doc/Documentation/CodingStyle

And Xerox PARC says camel case only:
http://www.ccnx.org/pubs/PARC-C-Style-Guide.pdf